express/manifest.py

```diff
@@ -163,16 +171,25 @@ def subsets(self) -> t.Mapping[str, Subset]:
             }
         )
 
+    def get_subset(self, name: str) -> Subset:
```

Comment:
Do we need this? `manifest.subsets[name]` should already provide the same behavior. I don't think the subsets list will become that long that the dict comprehension overhead matters.

I designed the manifest class following:
    - Fixed properties should be accessible as an attribute
    - Dynamic properties should be accessible by lookup

Reply:
Ok but that just returns a dictionary right, not a `Subset` instance? The latter allows to easily get properties like `location` and `fields`. Also you would get a `KeyError` in case the subset is not present in the manifest.

Reply:
Ok I've removed the `get_subset` method in favor of this